### DISCUSSION TITLE
Is there a way to search for a specific icon name in Excalidraw libraries?

### DISCUSSION THREAD

**Discussion by g1patil** [excalidraw/excalidraw#6870] [Q&A]
In the past had used few icons so did need search, wanted to know if there is a way to search for a icon from the added library ? Can't seem to find something that allows me to run a text search on the library.

**Comment by ad1992** [comment]
its in the roadmap, closing in favour of https://github.com/excalidraw/excalidraw/issues/6802

**Comment by mohsinmdl** [comment]
+1
